_posts/2022-08-05-nx-plugins.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Announcing our open-source project, Nx Plugins
+published: true
+description: Nx Plugins is a Nx mono repo which consists of plugins which supports the Nx eco-system.
+author: Farhan Patwary, Prabhat Thapa, Andrea Caldera, Danyal Aytekin
+usemathjax: true
+categories:
+  - nx-plugin
+  - end-to-end-testing
+tags:
+  - open-source
+  - nx
+  - playwright
+date: 2022-08-05
+banner:
+  image: terminal.png
```

Comment:
the path of this file is `assets/img/banners/terminal.png`. Does it just work by adding the file name like this?

Reply:
Yeah, I guess so. It is a template configuration for the blog here and I followed the same pattern. Once we will merge then we can check if this works 👍